pkg/sources/github/github.go

```diff
@@ -1109,6 +1148,39 @@ func (s *Source) chunkPullRequestComments(ctx context.Context, repo string, comm
 	return nil
 }
 
+// TODO: Should we resolve information like the Gist visibility? This will provide more context, but will exhaust ratelimits quicker.
+func (s *Source) chunkGistComments(ctx context.Context, comments []*github.GistComment, chunksChan chan *sources.Chunk) error {
+	for _, comment := range comments {
+		// Create chunk and send it to the channel.
+		chunk := &sources.Chunk{
+			SourceName: s.name,
+			SourceID:   s.SourceID(),
+			SourceType: s.Type(),
+			SourceMetadata: &source_metadatapb.MetaData{
+				Data: &source_metadatapb.MetaData_Github{
+					Github: &source_metadatapb.Github{
+						Link:     sanitizer.UTF8(comment.GetURL()),
+						Username: sanitizer.UTF8(comment.GetUser().GetLogin()),
+						Email:    sanitizer.UTF8(comment.GetUser().GetEmail()),
+						//Repository: sanitizer.UTF8(gist),
```

Comment:
Any reason `Repository` is commented out here?

Reply:
An oversight on my part. I've address this and the other points.